src/sentry/static/sentry/app/views/issueList/actions.tsx

```diff
@@ -176,36 +167,21 @@ type State = {
   selectedProjectSlug?: string;
 };
 
-const IssueListActions = createReactClass<Props, State>({
-  displayName: 'IssueListActions',
-
-  propTypes: {
-    api: PropTypes.object,
-    allResultsVisible: PropTypes.bool,
-    organization: SentryTypes.Organization.isRequired,
-    orgId: PropTypes.string.isRequired,
-    selection: SentryTypes.GlobalSelection.isRequired,
-    groupIds: PropTypes.instanceOf(Array).isRequired,
-    onRealtimeChange: PropTypes.func.isRequired,
-    onSelectStatsPeriod: PropTypes.func.isRequired,
-    realtimeActive: PropTypes.bool.isRequired,
-    statsPeriod: PropTypes.string.isRequired,
-    query: PropTypes.string.isRequired,
-    queryCount: PropTypes.number,
-  },
-
-  mixins: [Reflux.listenTo(SelectedGroupStore, 'handleSelectedGroupChange') as any],
```

Comment:
Just a note, you'll need to initialize component state with the store state, on mount